flow/flow.go

```diff
@@ -215,12 +215,15 @@ func New(options ...Option) *flowpb.Flow {
 		EventType:             EventType(),
 		SourceService:         Service(),
 		DestinationService:    Service(),
-		TrafficDirection:      flowpb.TrafficDirection(rand.Intn(3)),
+		TrafficDirection:      flowpb.TrafficDirection(rand.Intn(int(flowpb.TrafficDirection_EGRESS + 1))),
```

Comment:
I think we can actually use the map to make sure we also use the correct number.
```suggestion
		TrafficDirection:      flowpb.TrafficDirection(rand.Intn(len(flowpb.TrafficDirection_name) + 1)),
```